src/Bridges/Nette/TracyExtension.php

```diff
@@ -108,17 +108,20 @@ public function afterCompile(Nette\PhpGenerator\ClassType $class)
 
 		if ($this->debugMode) {
 			foreach ($this->config->bar as $item) {
-				if (is_string($item) && substr($item, 0, 1) === '@') {
-					$item = new Nette\DI\Statement(['@' . $builder::THIS_CONTAINER, 'getService'], [substr($item, 1)]);
-				} elseif (is_string($item)) {
-					$item = new Nette\DI\Statement($item);
+				if (is_string($item)) {
```

Comment:
why?

Reply:
Because in original implementation when you verify variable `$item` is string, second `elseif` condition does not make sense.